openlibrary/plugins/upstream/account.py

```diff
@@ -428,11 +428,11 @@ def POST(self):
                       token="", service="link")
         test = 'openlibrary' if i.token == lending.config_internal_tests_api_key else None
         if i.service == "link":
-            result = link_accounts(i.get('email').lower(), i.password,
-                                   bridgeEmail=i.bridgeEmail.lower(),
```

Comment:
In seeing the removal of these `lower()`s and the addition of them above, I wonder if it's worth adding some comments indicating why we want to have/not-have `lower()` in each location. Otherwise will be difficult to infer the context.